src/os/CMakeLists.txt

```diff
@@ -46,10 +46,10 @@ if(WITH_FUSE)
     FuseStore.cc)
 endif(WITH_FUSE)
 
-if(WITH_PMEM)
+if(WITH_BLUESTORE_PMEM)
   list(APPEND libos_srcs
     bluestore/PMEMDevice.cc)
-endif(WITH_PMEM)
+endif(WITH_BLUESTORE_PMEM)
```

Comment:
could you take this opportunity to remove the `WITH_BLUESTORE_PMEM` from `endif()`? this `if` block is very short.

Reply:
Got it. Done.